packages/firestore/exp-types/index.d.ts

```diff
@@ -68,7 +68,7 @@ export interface FirestoreDataConverter<T> {
   toFirestore(modelObject: Partial<T>, options: SetOptions): DocumentData;
   fromFirestore(
     snapshot: QueryDocumentSnapshot<DocumentData>,
-    options: SnapshotOptions
+    options?: SnapshotOptions
```

Comment:
Needed to make this interface compatible with the FirestoreDataConverter in lite.